sdk/src/main/java/co/omise/android/api/Config.kt

```diff
@@ -0,0 +1,58 @@
+package co.omise.android.api
+
+import android.os.Build
+import co.omise.android.BuildConfig
+
+/**
+ * Config class bundles configuration values supplied to the [Client] constructor.
+ *
+ * @param apiVersion The API version to use.
+ * @param publicKey  The key with the `pkey_` prefix.
+ *
+ * @see Client
+ */
+class Config(
+        private val apiVersion: String?,
+        private val publicKey: String?
```

Comment:
I think the constructor should not nullable.

Reply:
Same comment as above.

Reply:
👌